index.html

```diff
@@ -861,7 +861,7 @@ <h3 class="cmp-title__text">Join Thousands of Leaders Leaving a Lasting Impact</
                 <div id="container-fcf0395128" class="cmp-container">
                   <div  class="aem-Grid aem-Grid--12 aem-Grid--default--12">
                     <div class="aem-GridColumn aem-GridColumn--default--12 layout-grid">
-                      <div id="button-wrap" class="cmp-container">
+                      <div id="button-wrap2" class="cmp-container">

```

Comment:
Why aren't we using a class for this styling? It would go next to `layout-grid` and the CSS could specify `.button-wrap .cmp-container` or something. All these ids with appending a `2` at the end is not very component friendly.